shared/src/test/scala/org/mulesoft/yaml/DocRenderTest.scala

```diff
@@ -0,0 +1,18 @@
+package org.mulesoft.yaml
+
+import org.scalatest.funsuite.AnyFunSuite
+import org.scalatest.matchers.should.Matchers
+import org.yaml.model.YDocument
+import org.yaml.render.YamlRender
+
+class DocRenderTest extends AnyFunSuite with Matchers {
```

Comment:
YamlDocBuilderTest?

Reply:
I thought of that but that suite tests the DocBuilder specifically. I thought it wouldn't be 100% correct to put it there